.github/workflows/test.yaml

```
@@ -25,6 +25,24 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  ucxx-python-tests:
+    if: inputs.build_type == 'nightly'
+    steps:
+      - name: Checkout code repo
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
```

The latest version for this action is v4: https://github.com/actions/checkout